install_scripts/getSimId.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var exec = require('child_process').exec;
+var path = require('path');
+
+var SP = require('serialport');
+var SerialPort = SP.SerialPort;
+
+var privatePath = path.join(__dirname, '..', 'PRIVATE.json');
+
+exec('cat ' + privatePath + ' | grep -e \'"sim": "undefined"\'', function (err, stdout) {
```

Comment:
Finally, I think I'll do a require, modify the object, and rewrite it in the file, it'll be better.